web/packages/core/src/ruffle-player.ts

```diff
@@ -677,6 +685,17 @@ export class RufflePlayer extends HTMLElement {
         }
     }
 
+    /**
+     * Called when entering / leaving fullscreen
+     */
+    private fullScreenChange(): void {
+        this.instance?.set_fullscreen(
+            Boolean(
```

Comment:
Actually there already exists a `this.isFullscreen` property that does exactly this. Didn't notice it before...